dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt

```diff
@@ -181,7 +191,8 @@ private constructor(
 
         /**
          * Sets the environment name that will appear in your logs, traces and crash reports.
-         * This can be used to filter logs or traces and distinguish between your production and staging environment.
+         * This can be used to filter logs or traces and distinguish between your production
+         * and staging environment.
          * @param envName the environment name (default = "")
          */
         fun setEnvironmentName(envName: String): Builder {
```

Comment:
Shouldn't this be marked as deprecated now, as we expect `envName` in `Builder's` constructor?

Reply:
Not really sure...maybe. I will raise this in the daily